<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VDDHET5" failures="3" tests="8" name="test2" time="47.718" errors="0" timestamp="2025-07-01T10:12:35 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verifyAccountRegistration" time="2.375"/>
  <testcase classname="testCases.TC002_LoginTest" name="loginTest" time="1.752"/>
  <testcase classname="testCases.TC003_DataDrivenTest" name="dataDrivenTest" time="2.08"/>
  <testcase classname="testCases.TC003_DataDrivenTest" name="dataDrivenTest" time="1.662"/>
  <testcase classname="testCases.TC003_DataDrivenTest" name="dataDrivenTest" time="9.221">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_DataDrivenTest.dataDrivenTest(TC003_DataDrivenTest.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- dataDrivenTest -->
  <testcase classname="testCases.TC003_DataDrivenTest" name="dataDrivenTest" time="9.354">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_DataDrivenTest.dataDrivenTest(TC003_DataDrivenTest.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- dataDrivenTest -->
  <testcase classname="testCases.TC003_DataDrivenTest" name="dataDrivenTest" time="9.637">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_DataDrivenTest.dataDrivenTest(TC003_DataDrivenTest.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- dataDrivenTest -->
  <testcase classname="testCases.TC003_DataDrivenTest" name="dataDrivenTest" time="1.739"/>
</testsuite> <!-- test2 -->
